recipes/hepmc2/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "hepmc2" %}
+{% set version = "2.06.07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lcgapp.cern.ch/project/simu/HepMC/download/HepMC-{{ version }}.tar.gz
+  sha256: a0bdd6f36a3cc4cb59d6eb15cef9d46ce9b3739cae3324e81ebb2df6943e4594
+  folder: source
+  patches:
+    - patches/hepmc-2.03.06-reflex.patch
+    - patches/hepmc-2.06.07-WeightContainer-fix-size_type.patch
+    - patches/HepMC-2.06.07-nodoc.patch
+    - patches/HepMC-2.06.07-broken-tests.patch
+build:
+  number: 0
```

There should either be a `bld.bat` script for windows or it should be skipped. At the moment the windows build is an empty package.
```suggestion
  number: 0
  skip: True  # [win]
```